sections/production/installpackageswithnpmci.md

```diff
@@ -0,0 +1,30 @@
+# Install packages with npm ci in production
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+You locked your dependencies following [**Lock dependencies**](/sections/production/lockdependencies.md) but you now need to make sure those exact package versions are used in production.
+
+Using `npm ci` to install packages will do exactly that and more.
+* It will only install packages in the `dependencies` section of your `package.json`. `devDependencies` are ignored
```

Comment:
`npm ci` does install devDependencies?

Reply:
No. If you're asking, I guess the point is not clear enough! I'll try to rewrite that differently

Reply:
Sorry I meant to question the line, as `npm ci` does install devDependencies as well as dependencies.

It operates in the same way as npm install where you need to pass either the NODE_ENV variable as `production` or the `--production` cli flag

Reply:
Damn that's right, that's a really good catch. Thank you! I'll reword that part

Reply:
I'll create a new BP about using the --production flag in a new PR